docs/contributing/setting-up-your-local-dev-environment.md

```diff
@@ -57,6 +57,18 @@ Yarn is a package manager for your code, similar to [NPM](https://www.npmjs.com/
 
   - Note: if you plan to modify packages that are exported from `gatsby` directly, you need to either add those manually to your test sites so that they are listed in `package.json` (e.g. `yarn add gatsby-link`), or specify them explicitly with `gatsby-dev --packages gatsby-link`).
 
+- To revert the cloned code by `gatsby-dev` you can remove the `node_modules` directory or run:
+
+```shell
+git checkout package.json; yarn --force
+```
+
+or
+
+```shell
```

Comment:
Let's remove npm to keep these docs consistent on yarn as best practices.

Reply:
i changed it only in doc for consistency

Q: should i also remove this in readme? there is no explicit yarn there?

Reply:
I'd say we can leave it in the README since it's already there/may be accessed separately from the docs.